.github/workflows/app-samples-CI.yml

```diff
@@ -5,9 +5,9 @@ name: App samples CI
 
 on:
   push:
-    branches: [ main, hero_notes ]
+    branches: [ main, improve-ci ] # remove improve-ci before merging into main
```

Comment:
@CesarValiente we should remove this branch before the merge?

Reply:
yes, correct, goof catch!